starboard/tools/app_launcher_packager.py

```diff
@@ -114,12 +114,12 @@ def CopyAppLauncherTools(repo_root,
     additional_glob_patterns: Some platforms may need to include certain
       dependencies beyond the default include file patterns. The results here
       will be merged in with _INCLUDE_FILE_PATTERNS.
-    include_black_box_tests: If True then the resources for the black box tests
-      are included.
+    include_integration_tests: If True then the resources for the black box
```

Comment:
nit: consider changing the description to also say "integration tests"

Reply:
Done